.github/workflows/build-base-images.yml

```diff
@@ -57,26 +56,12 @@ jobs:
           username: ${{ github.repository_owner }}
           password: ${{ secrets.GITHUB_TOKEN }}
 
-      - name: Extract metadata (tags, labels) for Docker
-        id: meta
-        uses: docker/metadata-action@v4
-        with:
-          images: |
-            ghcr.io/${{ github.repository }}/${{ matrix.language }}
-          tags: |
-            type=sha
-            type=ref,event=tag
-            type=semver,pattern={{version}}
-            type=edge
-            type=raw,value=latest,enable={{is_default_branch}}
-
-      - name: "Build image"
+      - name: Build image and push
         id: image_build
-        uses: docker/build-push-action@v3
+        uses: devcontainers/ci@v0.2
         with:
-          push: true
-          context: "."
-          file: ./Dockerfiles/${{ matrix.language }}/Dockerfile
-          platforms: linux/amd64, linux/arm64
-          tags: ${{ steps.meta.outputs.tags }}
-          labels: ${{ steps.meta.outputs.labels }}
\ No newline at end of file
+          imageName: ghcr.io/${{ github.repository }}/${{ matrix.language }}
+          imageTag: latest
+          push: always
+          platform: linux/amd64,linux/arm64
+          subFolder: ./Dockerfiles/${{ matrix.language }}/
```

Comment:
Please fix all ```pre-commit run --all-files```  issues

Reply:
Added and will be pushed with the next commit